qiskit/__init__.py

```diff
@@ -46,6 +46,9 @@
 # Please note these are global instances, not modules.
 from qiskit.providers.basicaer import BasicAer
 
+# function to set user configuration
+from qiskit.user_config import set_config
```

Comment:
Maybe move these lines up to L33, so it is close to `_user_config`

Reply:
I would actually remove this from the `__init__.py`, this function doesn't need to be re-exported from the basis qiskit it's not likely to be a core function for users of qiskit like the other classes and functions we re-export. I think it makes more sense to only be off of `qiskit.user_config` so users would do:

```python3
from qiskit.user_config import set_config

set_config('num_processes', 1000000000)
```

it also is more clear what config we're talking about when we can only import it from `qiskit.user_config`